hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/SnapshotDeletingService.java

```diff
@@ -203,7 +225,8 @@ public BackgroundTaskResult call() throws Exception {
 
               for (OmKeyInfo keyInfo: repeatedOmKeyInfo.getOmKeyInfoList()) {
                 splitRepeatedOmKeyInfo(toReclaim, toNextDb,
-                    keyInfo, previousKeyTable);
+                    keyInfo, previousKeyTable, renamedKeyTable,
+                    bucketInfo, volumeId);
               }
```

Comment:
It should be fine as long as we are reading volume/bucket info directly from active DB. Since we don't allow deletion of buckets with snapshots in the first place. (And this is the important assumption here.)

Originally I thought we are going to get volume and bucket info from snapshot DB, in which case we would have to retain these two tables in `SstFilteringService` (which is [not the case](https://github.com/apache/ozone/blob/365f80dad239c3af799cd7c5794de697ede24cca/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SstFilteringService.java#L190-L197) at the moment).